challenges/2021-05-18-sum-of-squares/go.mod

```diff
@@ -0,0 +1,3 @@
+module solution
```

Comment:
I believe that we usually include the problem statement in comment at the top of implementation file.

Reply:
Ooh, yes, this!